app/posts/[identifier]/page.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { getMumblePostAction } from '@/actions/getMumblePost';
+import { PostCard } from '@/components/post-card/PostCard';
+import { PostEditor } from '@/components/post-editor/PostEditor';
+import PostActionsBar from '@/components/post-actions-bar/PostActionsBar';
+
+export default async function Page({
+  params,
+}: {
+  params: { identifier: string };
+}) {
+  const { identifier } = params;
+  const responseService = await getMumblePostAction(identifier, true);
+  const { postData, repliesData } = responseService;
+  return (
+    <div className="mr-auto ml-auto bg-slate-100 pt-8">
+      <div className="max-w-4xl mr-auto ml-auto py-8">
+        <div className="bg-white py-8 px-12 relative rounded-2xl mb-6">
+          <PostCard
+            text={postData.text}
+            id={postData.id}
+            creator={postData.creator}
+            mediaUrl={postData.mediaUrl}
+            mediaType={postData.mediaType}
+            likes={postData.likes}
+            replies={postData.replies}
+            likedBySelf={postData.likedBySelf}
+          />
+
+          <div className="mt-3">
+            <PostActionsBar
+              identifier={postData.id}
+              amountLikes={postData.likes}
+              amountComments={postData.replies}
+              selfLiked={postData.likedBySelf}
+            />
+          </div>
+          <div className="mt-3">
+            <PostEditor identifier={postData.id} />
+          </div>
+          {repliesData?.data?.map((dataReply, index) => {
+            return (
+              <React.Fragment key={index}>
```

Comment:
using the `index` for the key is considered a bad practise: https://react.dev/learn/rendering-lists#why-does-react-need-keys
Plz use the `dataReply.id` as a key.

Reply:
should be corrected by now.